src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Cohort.cs

```diff
@@ -658,5 +658,11 @@ public Party Approvals
                 return approvals;
             }
         }
+
+        public void TransferRequestRejectedReturnCohortToEmployer()
+        {
+            CheckIsWithParty(Party.TransferSender);
+            EditStatus = EditStatus.EmployerOnly;
```

Comment:
I seem to recall there being other properties we need to set. The cohort also has the `TransferApprovalStatus` property, and properties for employer email, name, and date. We should reproduce this behaviour.

Reply:
actually these other properties are only set when it's approved. When it's rejected these aren't updated